commands/command.go

```diff
@@ -217,12 +218,15 @@ func (c *Command) CheckArguments(req Request) error {
 	// iterate over the arg definitions
 	valueIndex := 0 // the index of the current value (in `args`)
 	for _, argDef := range c.Arguments {
-		// skip optional argument definitions if there aren't sufficient remaining values
-		if len(args)-valueIndex <= numRequired && !argDef.Required || argDef.Type == ArgFile {
+		// skip optional argument definitions if there aren't
+		// sufficient remaining values
+		if len(args)-valueIndex <= numRequired && !argDef.Required ||
+		argDef.Type == ArgFile {
```

Comment:
Jeromy Johnson <reply@reply.github.com> / 2016-02-02 21:30-0800
> > @@ -217,12 +218,15 @@ func (c *Command) CheckArguments(req Request) error {
> >  	// iterate over the arg definitions
> >  	valueIndex := 0 // the index of the current value (in `args`)
> >  	for _, argDef := range c.Arguments {
> > -		// skip optional argument definitions if there aren't sufficient remaining values
> > -		if len(args)-valueIndex <= numRequired && !argDef.Required || argDef.Type == ArgFile {
> > +		// skip optional argument definitions if there aren't
> > +		// sufficient remaining values
> > +		if len(args)-valueIndex <= numRequired && !argDef.Required ||
> > +		argDef.Type == ArgFile {
> 
> did you run gofmt? i'm not sure this is the right indentation here.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ipfs/go-ipfs/pull/2288/files#r51680222

gofmt++ (hadn't heard of it until now---what a noob, right?)